spec/openstudio/workflow/workflow_analysis_spec.rb

```diff
@@ -597,27 +597,97 @@
     expect(osw_out[:steps].size).to be > 0
     osw_out[:steps].each do |step|
       expect(step[:result]).to_not be_nil
+      # Only the EDA reporting measure step is supposed to have failed
+      if (step[:measure_dir_name] == "Xcel EDA Reporting and QAQC")
+        expect(step[:result][:step_result]).to eq 'Fail'
+      else
+        expect(step[:result][:step_result]).to eq 'Success'
+      end
     end
 
     expected_r = /Peak Demand timeseries \(Electricity:Facility at zone timestep\) could not be found, cannot determine the informati(no|on) needed to calculate savings or incentives./
     expect(osw_out[:steps].last[:result][:step_errors].last).to match expected_r
 
+    # TODO: Temporary comment
+    # Not sure why the in.idf ends up there at the root? Shouldn't it just be
+    # in run/in.idf?
```

Comment:
This should be deleted before merging. I'm mostly just confused about it and curious.